### PR TITLE
chore: setup package provenance

### PR DESCRIPTION
**Pull request by benmccann**

### CODE REVIEW

**Comment by WarningImHack3r** [comment]
@benmccann you duplicated the "publishConfig" key for the Cloudflare adapter (just so you know)

File: packages/adapter-cloudflare/package.json [hunk 1]
```diff
@@ -56,5 +56,8 @@
 	},
 	"publishConfig": {
```
Comment (benmccann) [inline packages/adapter-cloudflare/package.json:57]: you're on crack, github. this code doesn't exist: https://github.com/sveltejs/kit/blob/2b19df3855b13737399a7173f25e4fecbd7f689a/packages/adapter-cloudflare/package.json#L57

wtf is happening here??

Comment (Conduitry) [reply packages/adapter-cloudflare/package.json:57]: https://github.com/sveltejs/kit/commit/2b19df3855b13737399a7173f25e4fecbd7f689a#diff-1ac6ebb9247195e0dfa0f18975077643b0e0743c80cd8588b0084f5252cecec0 Changesets removed this when it bumped the version number for the release.

**Comment by benmccann** [comment]
okay, figured out the mystery. changesets automatically deleted the duplicate :scream: but that's fine because we didn't need the other one anyway. thanks for pointing this out!

**Comment by Conduitry** [comment]
Changesets presumably deleted the duplicate because `JSON.parse()` seems to silently ignore all but the last instance of a duplicated key. Thanks, javascript. That's an interesting choice.

**Pull request merged**